x-pack/plugins/uptime/public/components/monitor/status_details/location_map/embeddables/poly_layer_mock.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import lowPolyLayerFeatures from '../../low_poly_layer.json';
+import lowPolyLayerFeatures from './low_poly_layer.json';
```

Comment:
This file could stay in `__mocks__`. There is some special handling around mocks, like with code coverage reports.